### PR TITLE
 Finalzing Frontend

### PR DESCRIPTION
**Pull request by TcgVanguardTroll**
Finalizing the front end of portfolio

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -4,130 +4,40 @@
...
+            <p> I'm currently a rising junior at the University at Buffalo ! 
+                My major is computer science and my favorite classes so far have
+                been my intro courses as well as my data structures course.
+            </p>
+            <p> I am a 2x STEP inrern and I have worked on the Frameworks
+                web team where I worked alongside my podamte to work on 
+                create an internal debugging tool for internal google web
+                Frameworks.
+            </p>
+        </div>
```
Comment (kmh287) [inline portfolio/src/main/webapp/index.html]: indentation looks off here

**Review by kmh287 — APPROVED** [review]

File: portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (TcgVanguardTroll) [reply portfolio/src/main/webapp/index.html]: Ack

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -33,14 +33,27 @@ function addRandomFact(){
         'I am interested in compilers and progamming languages.',
         'I have interned at Google twice !',
         'I have lived in three states: California, New York, and Virginia.!',
+        'I enjoy writing and watching fantasy.',
+        'I am a memebr of NSBE.'
     ];
 
+    const factsTemp = [...facts];
```
Comment (tcwanggoogle) [inline portfolio/src/main/webapp/script.js:40]: Seems not used?

File: portfolio/src/main/webapp/index.html [hunk 3]
```diff
@@ -4,130 +4,40 @@
...
-    </ul>
-  </nav>
-
+    </div>
 
     <div id="content">
-      <h1>Jordan's Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p> I'm currently a rising junior at the University at Buffalo ! </p>
+            <h1>Welcome to my corner of the internet !</h1>
```
Comment (tcwanggoogle) [inline portfolio/src/main/webapp/index.html:21]: This indentation seems not right?

Comment (TcgVanguardTroll) [reply portfolio/src/main/webapp/index.html:21]: Ack

**Pull request merged**